ccpage171.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+# Add an attribute called number_served with a default value of 0.
+# Create an instance called restaurant from this class.
+# Add a method called set_number_served() that lets you set the number of customers that have been served.
+# Add a method called increment_number_served() that lets you increment the number of customers who’ve been served.
+
+class Restaurant():
+    """An attempt to model a restaurant"""
+    def __init__(self, restuaruant_name, restaurant_cuisine):
+        """Initialize the attributes"""
+        self.restuaruant_name = restuaruant_name
+        self.restuaruant_cuisine = restaurant_cuisine
+        self.number_served = 5
+    
+    def describe_restaruant(self):
+        """Write a variable that returns a summary of the restaurant"""
+        restaurant_details = self.restuaruant_name.title() + " serves " + self.restuaruant_cuisine.title() + " food."
+        return restaurant_details
+
+    # Add a method so I can set the number of guests served
+    def set_number_served(self, guests):
+        """Determine the number of people served at the restaurant"""
+        self.number_served = guests
+
+    # Add a method to increment the total number of people served
+    def increment_number_served(self, incremented_guests):
+        """Increment the total number of guests served"""
+        self.number_served += incremented_guests
+
+    # I added a method here so I can call the method, instead of printing a statment about the total number of guests served
+    def statement_for_number_served(self):
+        """Print a statement that tells the number of people served"""
+        print(f"The total number of people served is {int(self.number_served)}.")
```

Comment:
Nice job adding this method. `number_served` is already an `int`, so there's no need to cast (convert) it to a `int`.